client/src/components/App.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import axios from 'axios';
+import Namelist from './Namelist.jsx';
+
+class App extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            userPictures: [],
+            likes: 0,
+            likeWord: 'likes',
+
+
+        }
+        this.onClickLikes = this.onClickLikes.bind(this);
+        this.onClickProfiles = this.onClickProfiles.bind(this);
+
+    }
+
+    componentDidMount () {
+        axios.get('/userinfo')
+            .then( (data) => {
+                var array = [];
+                for(var i = 0; i<=8; i++){
+                    array.push(data.data[i])
+                }
+                this.setState({userPictures: array}, () =>{
+                    console.log(this.state.userPictures)
+                })
+                
+            })
+
+        axios.get('/likes')
+            .then ( (data) => {
+                var count = 0
+                data.data.forEach( (songObj) => {
+                    if(songObj.song_id === 1){
+                        count++
+                    }
+                })
+                this.setState({likes: count})
+                if(count === 1) {
+                    this.setState({likeWord: 'like'})
+                }
```

Comment:
Cool! I like that you keep likes and the word separate. Is it easier to have the likeWord on the state? Does it keep you from repeating the  if(count === 1) logic throughout your code?